CHANGELOG.rst

```diff
@@ -3,10 +3,16 @@ Changelog
 *********
 .. View rendered on https://fairrootgroup.github.io/FairCMakeModules/latest/changelog.html
 
-Unreleased
-==========
+Unreleased **v1.0.0** 2021-xx-xx
+================================
+
+New
+---
+
+* :command:`find_package2` auto-detects the install prefix and populates the result in
+  :variable:`<pkgname>_PREFIX`. Can be disabled globally via
+  :variable:`FAIR_NO_AUTO_DETECT_PREFIX`.
```

Comment:
I don't know, whether we need the global button now. I would vote for KISS. Add complexity, if we need it. Or if we have someone complain?

Reply:
Had the same thought first, but I want it in case we find some cases later when it is broken and then users may have already committed to FairCMakeModules in general and would hate to get stuff they do not need. Just a safety button so to say.

Reply:
Hmm, that would be the "complain" part.

Anyhow, looks good, is properly namespaced, etc. Resolving.